src/licensedcode/data/licenses/cornell-lossless-jpeg.LICENSE

```diff
@@ -0,0 +1,37 @@
+---
+key: cornell-lossless-jpeg
+short_name: Cornell Lossless JPEG License
+name: Cornell Lossless JPEG License
+category: Permissive
+owner: Kongji Huang
+spdx_license_key: Cornell-Lossless-JPEG
+other_urls:
+    - https://android.googlesource.com/platform/external/dng_sdk/+/refs/heads/master/source/dng_lossless_jpeg.cpp#16
+    - https://www.mssl.ucl.ac.uk/~mcrw/src/20050920/proto.h
+    - https://gitlab.freedesktop.org/libopenraw/libopenraw/blob/master/lib/ljpegdecompressor.cpp#L32
+ignorable_copyrights:
+    - Copyright (c) 1993 Cornell University, Kongji Huang
+ignorable_holders:
+    - Cornell University, Kongji Huang
+---
+
+Copyright (c) 1993 Cornell University, Kongji Huang
+All rights reserved.
+
+Permission to use, copy, modify, and distribute this software and its
```

Comment:
@pombredanne if we do this: `In the text here, let's replace "CORNELL UNIVERSITY" by "COPYRIGHT HOLDER"`
It would be exactly the same as `mit-modern`  which also is a SPDX license with: spdx_license_key: MIT-Modern-Variant. 
So we have to have two licenses here for SPDX detection correctness. And also requalify mit-modern rules.